jest.config.ts

```diff
@@ -1,30 +0,0 @@
-import path from 'path'
-import paths from './paths.config'
-
-export default {
-  preset: 'ts-jest',
-  testEnvironment: 'jsdom',
-  rootDir: paths.root,
-  roots: [
-    paths.app.src
-  ],
-  coverageThreshold: {
-    global: {
-      branches: 10,
-      functions: 42,
-      lines: 64,
-      statements: 60
-    }
-  },
-  collectCoverageFrom: [
-    '<rootDir>/src/**/*.{ts,tsx}'
-  ],
-  setupFilesAfterEnv: [
-    path.resolve(paths.config.jest, 'enzyme.setup.ts'),
-    path.resolve(paths.config.jest, 'jest.overrides.ts')
-  ],
-  moduleDirectories: [
-    'node_modules',
-    'src'
-  ]
-}
```

Comment:
why did u decide to remove all these files?

Reply:
they are duplicated )
these files should be in the `config` folder, but they were in root, and eslint was miss-intsructed about paths

Reply:
it wasn't possible to 'commit' until eslint errors were solved.